templates/CRM/Dashlet/Page/AllCases.tpl

```diff
@@ -23,10 +23,11 @@
  | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
  +--------------------------------------------------------------------+
 *}
-{if $AllCases}
-   <div class="form-item">
-       {include file="CRM/Case/Page/DashboardSelector.tpl" context="$context" list="allcases" rows=$AllCases}
-   </div>
+{if $casePresent}
```

Comment:
the logic is flawed here (and in MyCases.tpl). it only shows the filter if cases are present. however, that means you're locked into the default filters with no way to change it. so if, for example, there are no cases with upcoming activities in the next two weeks, you have no way to change the filter to show all cases. I think we can probably just remove the "no open cases"  clause altogether and persistently show the filter block.

Reply:
ok will remove this condtion.